.env.example

```diff
@@ -1,3 +1,3 @@
-BOT_USERNAME=
-BOT_PASSWORD=
-BOT_PREFIX=
+BOT_USERNAME=<facebook email/phone number/username>
+BOT_PASSWORD=<facebook password>
+BOT_PREFIX=<message identifier such as "@sadbot">
```

Comment:
That's why slash is probably the easiest option right?
/weather <city>
/dailyspace
etc, and agnostic with the botname